web/themes/joinup/joinup_theme.theme

```diff
@@ -1052,6 +1054,16 @@ function joinup_theme_theme_suggestions_container_alter(array &$suggestions, arr
       $suggestions[] = 'container__edit_actions__search_bar';
```

Comment:
I'm not sure this suggestion is necessary.
Also, `strpos($variables['element']['submit']['#id'], 'edit-submit-search') !== FALSE` could possibly a positive check to be shorter. Just please make sure this is actually needed at all.